Makefile.build

```diff
@@ -369,29 +369,55 @@ $(DOC_GRAM): $(DOC_GRAMCMO) coqpp/coqpp_parser.mli coqpp/coqpp_parser.ml doc/too
 	$(SHOW)'OCAMLC -a $@'
 	$(HIDE)$(OCAMLC) -I coqpp $^ -package str -linkall -linkpkg -o $@
 
-DOC_MLGS := */*.mlg plugins/*/*.mlg
-DOC_EDIT_MLGS := doc/tools/docgram/*_mlg
+# not included: plugins/ssr/ssrparser.mlg plugins/ssr/ssrvernac.mlg plugins/ssrmatching/g_ssrmatching.mlg
+DOC_MLGS := */*.mlg plugins/btauto/g_btauto.mlg \
```

Comment:
If you want to exclude those 3 files why not use filter-out instead?

Reply:
Got it.  I wanted something like that but I wasn't sure where to look for it.